doc/source/guide/spatial-partitioning.ipynb

```diff
@@ -0,0 +1,383 @@
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/ik19962/.local/lib/python3.8/site-packages/geopandas/_compat.py:111: UserWarning: The Shapely GEOS version (3.9.1-CAPI-1.14.2) is incompatible with the GEOS version PyGEOS was compiled with (3.10.1-CAPI-1.16.0). Conversions between both will be slow.\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "import geopandas\n",
+    "import dask_geopandas\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b6139e44-48d1-4906-b33a-eaa0c9b2bc3b",
+   "metadata": {},
+   "source": [
+    "We first begin by reading USA boundaries from gadm.org"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "a4d93451-bd17-4ca1-99c1-aed7f4d828b2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = \"https://geodata.ucdavis.edu/gadm/gadm4.0/gpkg/gadm40_USA.gpkg\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd57d7c7-91dd-4e09-937b-1707b7b2ce2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "usa = geopandas.read_file(path, layer=\"ADM_1\")\n",
+    "usa.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "247440da-ada6-4c25-bcd1-e1f0eb221d0c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Filter and rename cols\n",
+    "usa = usa[[\"NAME_1\", \"geometry\"]].rename(columns={\"NAME_1\": \"State\"})\n",
+    "usa.info()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20cdb8ab-f58c-436b-af5e-919f9f8b41a0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "usa.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c313f7bd-9748-4d8c-9e61-8bc112229f3e",
+   "metadata": {},
+   "source": [
+    "We can quickly filter for Contiguous United States, which represents the 48 joining states and the District of Columbia (DC) by the `cx` method.\n",
+    "This filters out any geometries using a bounding box."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ea92949a-7767-4d01-8541-e41ea2f71648",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "us_cont = usa.cx[-150:-50, 20:50]\n",
+    "us_cont.info()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "061cd916-390a-451d-ac46-5e273569c972",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "us_cont.plot(facecolor=\"none\", linewidth=0.5, edgecolor=\"red\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e4509aa4-4036-4621-ad0c-2e87eef2340a",
+   "metadata": {},
+   "source": [
+    "We can then transform the GeoDataFrame to a Dask one, with a fixed number of aribitrary partitions - this does not yet spatially partition a Dask GeoDataFrame.\n",
+    "Note: we can also read (and export) parquet files in Dask-GeoPandas containing spatially partitioned data."
```

Comment:
Use note admonition, like this (I think that the code suggestion would break this):
```
    ```{note} 
    We can also read (and export) parquet files in Dask-GeoPandas containing spatially partitioned data.
    ```
```